#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-version.dpatch by  <mhershberger@intrahealth.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix up the version checking

@DPATCH@

diff -ub Apache-Test-1.30/Makefile.PL\~ Apache-Test-1.30/Makefile.PL
--- Apache-Test-1.30/Makefile.PL~	2007-11-14 01:44:36.000000000 -0500
+++ Apache-Test-1.30/Makefile.PL	2008-01-29 21:56:49.000000000 -0500
@@ -2,7 +2,7 @@
 
 use lib qw(lib);
 use Apache::Test5005compat;
-
+use version;
 use strict;
 use warnings;
 
@@ -147,7 +147,7 @@
     }
 }
 
-if (TOP_LEVEL && $ExtUtils::MakeMaker::VERSION < 6.06) {
+if (TOP_LEVEL && version->new($ExtUtils::MakeMaker::VERSION) < version->new(6.06)) {
     # now that we're using subversion, make sure that
     # .svn directories are skipped during the build process
     # for old versions of MakeMaker
@@ -223,3 +223,4 @@
         $string;
     };
 }
+1

Diff finished.  Tue Jan 29 21:57:09 2008
